dll/win32/shell32/dialogs/view.cpp

```diff
@@ -342,10 +342,10 @@ ViewTree_LoadTree(HKEY hKey, LPCWSTR pszKeyName, DWORD dwParentID)
     else
     {
         // HKeyRoot
-        Value = DWORD(HKEY_CURRENT_USER);
+        Value = PtrToUlong(HKEY_CURRENT_USER);
```

Comment:
I think this is more broken than just the cast. HKEY_CURRENT_USER has the upper 32 bits set (at least according to our psdk), so this will truncate it.